forch/__main__.py

```diff
@@ -59,6 +59,7 @@ def run_forchestrator():
         http_server.map_request('process_state', forchestrator.get_process_state)
         http_server.map_request('host_path', forchestrator.get_host_path)
         http_server.map_request('list_hosts', forchestrator.get_list_hosts)
+        http_server.map_request('vrrp_state', forchestrator.get_vrrp_state)
```

Comment:
I think this is too small to be its own top-level API... it should be part of system_state  (unless there's a really good reason it needs to be separate).

Reply:
So this was actually Eng's request. They would like VRRP state to be a subsystem like the others, and to display the detailed vrrp error through vrrp_state API instead of the system_state_detail.

Reply:
Ok -- just something to be careful of (in terms of breaking encapsulations), but sounds like a reasonable enough ask!